packages/canary-bot/src/canary-bot.ts

```diff
@@ -40,6 +42,14 @@ const cronIssueTitle = 'A canary is chirping';
 const myRepositoryName = 'repo-automation-bots';
 const myOrganizationName = 'googleapis';
 
+interface PubSubContext {
+  github: Octokit;
+  readonly event: string;
+  log: Logger;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  payload: any;
```

Comment:
Would it be possible to make this interface `<T = any>` so consumers can attach a type to it?

Reply:
Sure